src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -53,6 +53,18 @@ function buildDeck(playerColor, deckId)
       loadCards)
 end
 
+-- Modelled after onDeckResult from arkhamdb/ArkhamDb.ttslua, but with some necessary changes
+function spawnDeckFromToken(params)
+  local deckData = JSON.decode(params.coin.getGMNotes())
+  destroyObject(params.coin)
+  deckData["playerColor"] = params.playerColor
+  -- Add Investigator Cards
+  deckData["slots"][deckData.investigatorId] = 1
+  deckData["slots"][string.sub(deckData.investigatorId, 1, 5) .. "-m"] = 1
```

Comment:
Can you explain why this is not part of the original saved deck data?

Reply:
In order to be able to import the deck to the deck builder.  The deck builder looks at the slots as well and populates a deck with it.  Don't want the investigator card included in that.  It needs to be to be compatible with the ArkhamDB code though, so it's added in manually here.

Reply:
I think it's easier just to add it in here.  In order to exclude it on the deck builder I'd have to either search through the list or add a check into a for loop.  This on the other hand is an instruction that gets executed once.  I feel like it's preferable.